### PR TITLE
Stop claiming easy_install support.

### PR DESCRIPTION
**Pull request by ni-jfitzger**
- [X] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).
- [X] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.
- ~[ ] I've added tests applicable for this pull request~.

### What does this Pull Request accomplish?
- Remove mentions of deprecated easy_install option.
- [Setuptools deprecated the easy_install command in 2021](https://setuptools.pypa.io/en/latest/history.html#v58-3-0)

### List issues fixed by this Pull Request below, if any.
None

### What testing has been done?
None

### CODE REVIEW

File: generated/nifgen/README.rst [hunk 1]
```diff
@@ -114,12 +114,6 @@ The nimi-python modules (i.e. for **NI-FGEN**) can be installed with `pip <http:
 
   $ python -m pip install nifgen
 
-Or **easy_install** from
-`setuptools <http://pypi.python.org/pypi/setuptools>`_::
-
-  $ python -m easy_install nifgen
-
-
```
Comment (ni-jfitzger) [inline generated/nifgen/README.rst:122]: Should I have left another blank line?

Comment (marcoskirsch) [reply generated/nifgen/README.rst:122]: I think so

**Review by marcoskirsch — CHANGES_REQUESTED** [review]

**Comment by marcoskirsch** [comment]
Some system tests failed, please check why.

File: CHANGELOG.md [hunk 2]
```diff
@@ -36,6 +36,7 @@ All notable changes to this project will be documented in this file.
     * #### Added
     * #### Changed
     * #### Removed
+        * `easy_install` support dropped.
```
Comment (marcoskirsch) [inline CHANGELOG.md:39]: FWIW, this is under "Removed" which makes it redundante to say "dropped" in the line item.

**Pull request merged**